source/globalCommands.py

```diff
@@ -3675,6 +3675,18 @@ def _enableScreenCurtain(doEnable: bool = True):
 			else:
 				_enableScreenCurtain()
 
+	@script(
+		description=_(
+			# Translators: Describes a command.
+			"Toggles through paragraph navigation styles",
+		),
+	)
+	def script_toggleParagraphStyle(self, gesture: "inputCore.InputGesture") -> None:
+		from utils.paragraphHelper import nextParagraphStyle
+		newFlag: config.featureFlag.FeatureFlag = nextParagraphStyle()
+		config.conf["documentNavigation"]["paragraphStyle"] = newFlag

```

Comment:
This needs to be a string

```suggestion
		config.conf["documentNavigation"]["paragraphStyle"] = newFlag.name
```

Reply:
Is this still an issue after resetting your config? It looks like an old invalid value for the feature flag is set

Reply:
are you setting the config value in a different place?

Reply:
Yes, my config has been reset. After resetting, I switched to block style, resulting in:
paragraphStyle = BLOCK
By the way, is this supposed to be stored as uppercase in the config?

The pattern for the crash is as follows:
1. I have a profile for Notepad++, wich is basically empty. It does not have a setting for paragraphStyle.
2. Run development version of NVDA.
3. Write a valid string to config.conf["documentNavigation"]["paragraphStyle"]. This can be done either by changing the setting in the Settings panel, or toggling through the different styles. Also, writing the value from the NVDA Console will work.
4. Switch to Notepad++.
5. Switch away from Notepad++.
6. At that point, the above error is given on every keypress, and NVDA never focuses on the active application.
7. If config.conf["documentNavigation"]["paragraphStyle"] is never written to, everything functions normally.

Reply:
This looks like a bug with `FeatureFlag`. We'll aim to fix this as a priority to unblock this PR.

Reply:
This has been fixed on this PR https://github.com/nvaccess/nvda/pull/14128